delivery/schemas.py

```diff
@@ -12,3 +12,16 @@ class RestaurantCreateOrUpdateSchema(Schema):
     @post_load
     def make_restaurant(self, content, **kwargs):
         return Restaurant(**content)
+
+
+class UserSchema(Schema):
+    login = fields.Str(required=True)
```

Comment:
`Schema` is for request/response parsing. `password` should never be exposed to the outer world in response)

Reply:
fixed